lib/Controller/LogController.php

```diff
@@ -1944,8 +1944,12 @@ public function logGpsloggerPost(string $token, string $devicename, float $lat,
 	 * @param float|null $batt
 	 * @return mixed
 	 */
-	public function logOwntracks(string $token, float $lat, float $lon, ?string $devicename = null, ?string $tid = null,
+	public function logOwntracks(string $token, ?float $lat, ?float $lon, ?string $devicename = null, ?string $tid = null,
```

Comment:
Why not giving a default null value to lat and lon? It would make it more explicit that we handle the absence of those params.

Reply:
Not quite sure about this as the main purpose of this function is to log lat/lon and those being "null" should be the exception.

Reply:
I agree.
But what if, at some point, Symfony changes its behaviour when a GET parameter is missing and passes something else than null to the controller method?
Anyway, not a big deal. Good as it is now. Thanks!